roles/wordpress-install/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- include: directories.yml
+
+- name: Create .env file
+  template: src="env.j2"
+            dest="{{ www_root }}/{{ item.key }}/current/.env"
+            owner="{{ item.value.user }}"
+            group="{{ item.value.group }}"
```

Comment:
Good catch, I removed these in a few other places but missed this one. I'll do a search for any other occurrences.